roles/ceph-defaults/tasks/facts.yml

```diff
@@ -216,7 +216,7 @@
     ceph_uid: 167
   when:
     - containerized_deployment
-    - ceph_docker_image_tag | string is search("latest") or ceph_docker_image_tag | string is search("centos") or ceph_docker_image_tag | string is search("fedora")
+    - ceph_docker_image_tag | string is search("centos") or ceph_docker_image_tag | string is search("fedora")
```

Comment:
```suggestion
    - ceph_docker_image_tag | string is search("latest-") or ceph_docker_image_tag | string is search("centos") or ceph_docker_image_tag | string is search("fedora")
```

Reply:
Will, that solve your problem?

Reply:
@leseb Works for me. We are using ``luminous-latest`` and ``latest`` as our tags.